packages/samples/atomic-react/src/App.tsx

```diff
@@ -1,218 +1,40 @@
-import React, {FunctionComponent} from 'react';
+import React, {FunctionComponent, useState} from 'react';
```

Comment:
It was there before your PR, but if you can find the magic recipe for the linter config to suppress that warning, it would be awesome.